### ISSUE TITLE
Multiple-file selection in lighttable-mode doesn't work

### ISSUE THREAD

**Issue by rand00** [darktable-org/darktable#12037]
Did you buy darktable from an application store ?
No

Describe the bug/issue
I can't select multiple files with left-mouse button + shift/ctrl in lighttable mode. I can use Ctrl-A though, and I can select single images with left-mouse button.

To Reproduce
Already described

Expected behavior
Able to select multiple images - I used to be able to do this 

Which commit introduced the error
I tried to compile darktable locally but get error:
```bash
CMake Warning at CMakeLists.txt:314 (find_package):
  Could not find a configuration file for package "LLVM" that is compatible
  with requested version "14".

  The following configuration files were considered but not accepted:

	/usr/lib64/cmake/llvm/LLVMConfig.cmake, version: 13.0.1
	/usr/lib/cmake/llvm/LLVMConfig.cmake, version: 13.0.1
	/lib64/cmake/llvm/LLVMConfig.cmake, version: 13.0.1
	/lib/cmake/llvm/LLVMConfig.cmake, version: 13.0.1

Call Stack (most recent call first):
  CMakeLists.txt:322 (find_llvm)
```

Platform

* darktable version : 3.8.1
* OS                :  Linux 5.18.3-arch1-1 
* Linux - Distro    : Arch Linux
* Memory            :  16Gb
* Graphics card     :  Geforce GTX 970
* Graphics driver   : nvidia 515.48.07-9
* OpenCL installed  : 
* OpenCL activated  : 
* Xorg              : 
* Desktop           : 
* GTK+              : 
* gcc               : 
* cflags            : 
* CMAKE_BUILD_TYPE  :

**Comment by dterrahe** [comment]
@Nilvus mouse clicks in the central view have very little to do with InputNG directly (only alternative modifier keys, on midi devices etc) and nothing comes to mind that was fixed/changed in 3.8.1.

@rand00 and @caigner these kind of reports are the hardest for a developer to work on, because if we cannot reproduce it on our own machine, we cannot figure out what change might have impacted it and what fix would work. Unfortunately we are fully dependent on you to help with further diagnosing and testing.

The first thing I would like to ascertain is that the problem is really in the dt code. For that I would ask that you compile from source not just the latest version (which exhibits the problem) but also 3.8.0 which, if installed from the repository _doesn't_ have the problem. If the from source compiled version of 3.8.0 is _also_ problem free, then indeed dt is suspect. It is however possible that a change in another package (gtk is always a likely candidate, since we use it so intensively and not always in ways that the maintainers would expect or test) _only_ causes a problem when compiled with it, but not during runtime if dt was compiled against a previous version (as the repository version of 3.8.0 likely was). This would be the case if the bug was introduced in an inlined function, for example.

If the problem _does_ come from dt, a `git bisect` result would be really useful, so we can see precisely which commit introduced the problem and we can then start experimenting with fixes. This could end up being purely guesswork, since presumable any such "fixes" won't make a difference on our already working systems...

**Comment by Nilvus** [comment]
Thanks @dterrahe for your precise and detailed answer. I see that @AlicVB answer and @caigner issue with 2 commits. I suggest to continue in @caigner issue #11927 to avoid confusion.

@rand00: pleasexreport here if you fix llvm error on build. For select issue report back on #11927 issue. Then we can close this one to avoid duplicate issue.

**Comment by elstoc** [comment]
> Though now I have a problem compiling release-3.8.1

This is a separate issue from what we're discussing, which can be patched (see https://github.com/darktable-org/darktable/commit/c5a3ca02580799bdabdd767b876445dc19e7bb4a), or you can wait for release-4.0, in which it is resolved.

**Comment by rand00** [comment]
> This is a separate issue from what we're discussing,

@elstoc I'm not of that impression - see what @dterrahe wrote:

> If the from source compiled version of 3.8.0 is _also_ problem free, then indeed dt is suspect.

I'll try to apply the patch

**Comment by rand00** [comment]
I manually fixed another compilation problem on 3.8.0, and got it built. 

The UI problems are not present in 3.8.0. So selection of multiple images works again + switch window focus doesn't stop shortcuts from working. 

I'll try to bisect in the evening if I don't hit the same amount of compilation problems on other versions. Is there a flag I can pass to ignore minor warnings so I can be sure dt compiles while bisecting?

**Comment by Nilvus** [comment]
@rand00: for selection issue, as I said, please continue in #11927 issue now. Copy-paste your last comment on #11927 also. Thanks. It will be hard for devs to follow 2 issues. So as you succeed to compile, I close this one.
